pkgs/default.nix

```diff
@@ -148,8 +148,9 @@ in rec {
 
   get-user-data = pkgs.writeScriptBin "get-user-data" ''
     #! /bin/sh
-    exec ${retry}/bin/retry \
-      ${pkgs.curl}/bin/curl -sf http://169.254.169.254/latest/user-data
+    exec ${pkgs.curl}/bin/curl \
```

Comment:
but curl's retry sucks, e.g. wget is used here because of that: https://github.com/zalora/microgram/blob/3cb4cbb3d12742e042c2846c0c1b7f83c4c812db/nixos/ec2/default.nix#L26